packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3234,9 +3234,16 @@ class EditableTextState extends State<EditableText>
   void didChangeDependencies() {
     super.didChangeDependencies();
 
-    _style = MediaQuery.boldTextOf(context)
-        ? widget.style.merge(const TextStyle(fontWeight: FontWeight.bold))
-        : widget.style;
+    // Apply platform settings to text style.
+    final ui.TypographySettings? typographySettings = MediaQuery.maybeTypographySettingsOf(context);
+    _style = widget.style.merge(
+      TextStyle(
+        height: typographySettings?.lineHeight,
```

Comment:
Is this lineHight in px? I thought this can be unitless and be something like double the existing height..etc

Reply:
Changed this `lineHeight` to be a factor of font size similar to what `TextStyle.height` is, this makes more sense.